apps/cli/src/commands/doctor.ts

```diff
@@ -0,0 +1,47 @@
+import { Command, Flags } from '@oclif/core';
+import { execSync } from 'child_process';
+
+export default class DoctorCommand extends Command {
+  static description = 'Verify the minimal requirements for the sunodo execution commands';
+
+  static examples = ["<%= config.bin %> <%= command.id %>"];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+  };
+
+  public async run() {
+    try {
+      // Check Docker version
+      this.log('Checking Docker installation...');
+      const dockerVersionOutput = execSync('docker -v').toString().trim();
+      const dockerVersion = dockerVersionOutput.match(/(\d+\.\d+\.\d+)/)?.[0] || 'Unknown';
+      this.log(`Docker version ${dockerVersion} is installed`);
+
+      // Check Docker Compose version
+      this.log('Checking Docker Compose installation...');
+      const dockerComposeVersionOutput = execSync('docker-compose -v').toString().trim();
```

Comment:
Use `docker compose version --format json`
We need `docker compose`, which is the compose plugin, not the old `docker-compose`

Reply:
Fixed ! 

We just need to define minimum docker compose plugin version